ReflectorIOS/Extensions/UINavigationController+Maroon.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  UINavigationController+Maroon.swift
+//  ReflectorIOS
+//
+//  Created by Nikhil Menon on 9/14/20.
+//  Copyright © 2020 Nikhil Menon. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+extension UINavigationController {
+    override open func viewDidLoad() {
+        super.viewDidLoad()
+      
+        let appearance = UINavigationBarAppearance()
+        appearance.backgroundColor = Theme.reflectorMaroon
+        appearance.titleTextAttributes = [NSAttributedString.Key.foregroundColor:UIColor.white]
```

Comment:
Add spacing after the colon... `[NSAttributedString.Key.foregroundColor: UIColor.white]`